internal/runtime/helper.go

```diff
@@ -37,8 +38,9 @@ func constructTransactions(
 	fmt.Printf("\n⏳ Estimating Gas ⏳\n")
 
 	// Estimate the fee for the transaction batch
+	// passing in the maximum block gas, this is just a simulation
 	txFee := common.CalculateFeeInRatio(
-		1_000_000,
+		types.MaxBlockMaxGas,
```

Comment:
This shouldn't be a constant value, but the block gas limit value of the latest block on the chain we're trying to run the stress test against